evm/src/cpu/kernel/asm/memory.asm

```diff
@@ -26,6 +26,47 @@
     // stack: (empty)
 %endmacro
 
+// Load a single byte from kernel code.
+%macro mload_kernel_code
```

Comment:
Just curious, would it be more "idiomatic" to make this take `offset` as an argument? Or maybe that's an undefined notion haha

Reply:
Hm I don't think we can with the current asm syntax; for now only int literals are allowed as macro args.